### ISSUE TITLE
Running under Gnome-SDK-3.16 fails, advice?

### ISSUE THREAD

**Issue by AsavarTzeth** [jonls/redshift#196]
I have been hacking with the xdg-app and the Freedesktop & Gnome runtimes. I choose redshift as a simple test application, since it is among my favorites. Everything builds well and I am happy to share my work if it is deemed useful.

However, when I run the application I get several errors. Some, or perhaps even all of these are not related to redshift. But I have no way of being sure since I don't know python.

Here is the full traceback:

```
(process:8): Gtk-WARNING **: Locale not supported by C library.
    Using the fallback 'C' locale.
Gtk-Message: Failed to load module "canberra-gtk-module"
No protocol specified

** (redshift-gtk:8): WARNING **: Could not open X display
No protocol specified
Traceback (most recent call last):
  File "/self/bin/redshift-gtk", line 26, in <module>
    from redshift_gtk.statusicon import run
  File "/self/lib/python3.3/site-packages/redshift_gtk/statusicon.py", line 40, in <module>
    from . import utils
  File "/self/lib/python3.3/site-packages/redshift_gtk/utils.py", line 20, in <module>
    import ctypes
  File "/usr/lib/python3.3/ctypes/__init__.py", line 7, in <module>
    from _ctypes import Union, Structure, Array
ImportError: No module named '_ctypes'
```

The bundled python version is 3.3 & all core dependencies should be there. Does anyone have any interest in helping me figure this out? I have already contacted the main developer from the Gnome project in question but I don't want to waste his time too much before I know more.

Please tell, if you need any more information. I am grateful for any assistance.

**Comment by jonls** [comment]
This does not look like an issue with Redshift.

**Comment by AsavarTzeth** [comment]
Okay thank you for confirming what I already suspected. I did not know where to ask (and I guess I still don't).

Any idea what could cause it? To me it looks like the module is partly missing or broken, but my knowledge is limited. I already suggested this to the Gnome dev. But so far he simply insist I need to bundle everything that is missing. The thing is all major distributions I have used include a working ctypes module with their python package.

Anyways if you want me to close this and look elsewhere just tell me.
